ml-conversational-analytic-tool/githubDataExtraction.py

```diff
@@ -161,17 +155,26 @@ def getPullFeatures(self, pull):
     parser = argparse.ArgumentParser(description='Create csv for all pulls in repo')
     parser.add_argument('organization', help='Organization the repo belongs to.')
     parser.add_argument('reponame', help='Name of repo')
-    parser.add_argument('-reactions', action='store_true', default=False, help='Flag to extract reactions')
-    parser.add_argument('--filename', help='Name of file')
-    # parser.add_argument('accesstoken', help='Github access token')
+    parser.add_argument('--reactions', action='store_true', default=False, help='Flag to extract reactions')
```

Comment:
`--reaction` used to be with a single dash. If you want to keep double dash need to update the README file as well.

Reply:
I think we should keep it double dash and update the README to reflect that it's double dash instead of one.

Reply:
I edited the README accordingly.